applications/party/src/main/java/org/apache/ofbiz/party/party/PartyServices.java

```diff
@@ -1604,14 +1604,17 @@ public static Map<String, Object> performFindParty(DispatchContext dctx, Map<Str
                     EntityFunction.upper("%" + partyId + "%")));
         }
 
-        // now the statusId - send ANY for all statuses; leave null for just enabled; or pass a specific status
+        // now the statusId - null for all statuses; "PARTY_ENABLED" finds statusId == null entries also, other passed in values are matched exactly
         if (UtilValidate.isNotEmpty(statusId)) {
-            andExprs.add(EntityCondition.makeCondition("statusId", statusId));
-        } else {
-            // NOTE: _must_ explicitly allow null as it is not included in a not equal in many databases... odd but true
-            andExprs.add(EntityCondition.makeCondition(EntityCondition.makeCondition("statusId", GenericEntity.NULL_FIELD),
-                    EntityOperator.OR, EntityCondition.makeCondition("statusId", EntityOperator.NOT_EQUAL, "PARTY_DISABLED")));
+            if ("PARTY_ENABLED".equals(statusId)) {
```

Comment:
Could you please share use case why this change is needed?

Reply:
I think it's OK to be able to select disabled party. That can make sense in some situation.

Reply:
We'll discuss those changes and will provide an altered pull request. 
Will take at least two weeks as georg is not available atm.